src/Explore/DocumentItem.js

```diff
@@ -22,81 +31,144 @@ function DocumentItem(props) {
+              }),
             }}
           >
-            <FiExternalLink />
-            {doi}
-          </Box>
-          <ScoringIndicator score={score} />
-        </Flex>
-        <Heading
-          onMouseEnter={() => setIsHovered(true)}
-          onMouseLeave={() => setIsHovered(false)}
-          as={Link}
-          href={doiLink}
-          target="_blank"
-          sx={{
-            display: 'block',
-            whiteSpace: isDetailed ? 'wrap' : 'nowrap',
-            overflow: isDetailed ? 'visible' : 'hidden',
-            textOverflow: isDetailed ? 'none' : 'ellipsis',
-            textDecoration: isHovered ? 'underline' : 'none',
-          }}
-        >
-          {title}
-        </Heading>
-        <Box
-          onClick={() => isDetailed || setIsDetailed(true)}
-          sx={{
-            mx: -3,
-            mb: -3,
-            pb: 3,
-            px: 3,
-            pt: 1,
-            ':hover': {
-              bg: isDetailed || 'foreground',
-              cursor: isDetailed || 'pointer',
-            },
-          }}
-        >
-          {isDetailed ? <Detailed {...document} /> : <Simple {...document} />}
+            {title}
+          </Heading>
           <Box
-            onClick={() => setIsDetailed(false)}
-            sx={{
-              cursor: 'pointer',
-              textAlign: 'center',
-              fontSize: 4,
-              display: isDetailed ? 'block' : 'none',
-              mx: -3,
-              mb: -3,
-              ':hover': {
-                bg: 'foreground',
-              },
-            }}
+            sx={
+              !isDetailed && {
+                display: '-webkit-box',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                '-webkit-line-clamp': '2',
+                '-webkit-box-orient': 'vertical',
+              }
+            }
           >
-            <FiChevronUp />
+            {summary}
           </Box>
         </Box>
+        {!isDetailed && (
+          <Flex
+            sx={{
+              flexDirection: ['column', 'row'],
+              alignItems: ['flex-start', 'center'],
+              fontSize: [0, 0, 1],
+              mt: 1,
+              py: 2,
+            }}
+          >
+            <Button
+              variant="base"
+              sx={{
+                flex: 'none',
+                py: 1,
+                ':hover': { textDecoration: 'underline' },
+              }}
+              onClick={(evt) => {
+                setIsDetailed(true)
+              }}
+            >
+              <FiChevronRight />
+              <Text as="span" ml={2}>
+                Details, services ...
+              </Text>
+            </Button>
+            {(releaseDate || provider) && (
+              <Text
+                as="p"
+                sx={{
+                  m: 0,
+                  my: [2, 0],
+                  flex: '1 1 0%',
+                  textAlign: 'right',
+                  order: [-1, 0],
```

Comment:
Putting the "released on" text above the "Details, services ..." button on mobile.